stetho/src/main/java/com/facebook/stetho/common/android/FragmentActivityAccessor.java

```diff
@@ -6,10 +6,9 @@
 
 import javax.annotation.Nullable;
 
-public interface FragmentActivityAccessor {
+public interface FragmentActivityAccessor<
+    FRAGMENT_ACTIVITY extends Activity,
+    FRAGMENT_MANAGER> {
   @Nullable
-  public Object getFragmentManager(Activity fragmentActivity);
-
-  @Nullable
-  public Object getSupportFragmentManager(Activity fragmentActivity);
+  public FRAGMENT_MANAGER getFragmentManager(FRAGMENT_ACTIVITY activity);
```

Comment:
I'm pretty sure we can't remove the getSupportFragmentManager() method from this interface. The reason is that, at run-time, we could have an Activity class deriving from the v4 Support FragmentActivity. In this case we would have both getFragmentManager() and getSupportFragmentManager() and we do need to check both of them.